### DISCUSSION TITLE
SwaggerHub and PostgREST API Service

### DISCUSSION THREAD

**Discussion by mchatrvd** [PostgREST/postgrest#3039] [Q&A]
Hi I was able to install the PostgREST service and am able to call the API and extract data from PostgreSQL database in the JSON format. I am trying to set this up in SwaggerHub, so that users can use SwaggerHub to invoke and use the API. Any idea how can i go about setting this up in SwaggerHub. It seems like little bit of reverse engineering where we have  working API but need to set that up in SwaggerHub. Any help is appreciated. Thanks in advance.

**Comment by laurenceisla** [comment]
Hi. The [OpenAPI output](https://postgrest.org/en/stable/references/api/openapi.html) is at the root path, e.g. `http://localhost:3000/`. Not sure how to integrate it with SwaggerHub though.
